ivre/tools/ipdata.py

```diff
@@ -92,21 +94,23 @@ def my_parse_args():
                 ivre.db.db.data.ensure_indexes()
     if args.download:
         ivre.geoiputils.download_all(verbose=not args.quiet)
+    dbtofeed = {}
+    if not args.dont_feed_ipdata_cols:
+        dbtofeed["feedipdata"] = [ ivre.db.db.passive ]
+        if args.update_nmap_db:
```

Comment:
This one should probably be outside the `if not args.dont_feed_ipdata_cols:`: we may want to use both `--update-nmap-db` **and** `--dont-feed-ipdata-cols`, or do you think it does not make sense?

```python
dbtofeed = {}
if not args.dont_feed_ipdata_cols:
    dbtofeed["feedipdata"] = [ivre.db.db.passive]
if args.update_nmap_db:
    dbtofeed.setdefault("feedipdata", []).append(ivre.db.db.passive)
```

Also, please try to be [PEP8](https://www.python.org/dev/peps/pep-0008/)-compliant (no space inside brackets).